rust/azure_iot_operations_services/src/azure_device_registry/client.rs

```diff
@@ -735,6 +786,62 @@ where
         }
     }
 
+    /// Creates or updates a discovered device in the Azure Device Registry service.
```

Comment:
Can you add some information about how to use this for an update? Something like:
To perform an update, the entire existing device specification plus changes must be included. This operation will do a full replace